assets/xml/objects/object_ru2.xml

```diff
@@ -1,78 +1,97 @@
+        <Texture Name="gAdultRutoHeadHoleTex" OutName="adult_ruto_head_hole" Format="ci8" Width="8" Height="8" Offset="0x1960" TlutOffset="0xE00"/>
+        <Texture Name="gAdultRutoMouthTex" OutName="adult_ruto_mouth" Format="rgba16" Width="32" Height="32" Offset="0x19A0"/>
+        <Texture Name="gAdultRutoTex_21A0" OutName="adult_ruto_tex_21A0" Format="ci8" Width="8" Height="8" Offset="0x21A0" TlutOffset="0xE00"/>
+        <Texture Name="gAdultRutoEar2Tex" OutName="adult_ruto_ear_2" Format="ci8" Width="16" Height="16" Offset="0x21E0" TlutOffset="0xE00"/>
+        <Texture Name="gAdultRutoEarringTex" OutName="adult_ruto_earring" Format="rgba16" Width="8" Height="16" Offset="0x32E0"/>
+        <Texture Name="gAdultRutoTailGradientTex" OutName="adult_ruto_tail_gradient" Format="ci8" Width="8" Height="8" Offset="0x45C0" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoTex_4600" OutName="adult_ruto_tex_4600" Format="ci8" Width="8" Height="8" Offset="0x4600" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoBackOfHandTex" OutName="adult_ruto_back_of_hand" Format="ci8" Width="16" Height="16" Offset="0x4640" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoTex_4740" OutName="adult_ruto_tex_4740" Format="ci8" Width="16" Height="16" Offset="0x4740" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoSkinPattern1Tex" OutName="adult_ruto_skin_pattern_1" Format="ci8" Width="32" Height="64" Offset="0x4840" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoSkinPattern2Tex" OutName="adult_ruto_skin_pattern_2" Format="ci8" Width="32" Height="32" Offset="0x5040" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoTex_5440" OutName="adult_ruto_tex_5440" Format="ci8" Width="8" Height="8" Offset="0x5440" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoTex_5480" OutName="adult_ruto_tex_5480" Format="ci8" Width="16" Height="16" Offset="0x5480" TlutOffset="0x43C0"/>
+        <Texture Name="gAdultRutoTex_5580" OutName="adult_ruto_tex_5580" Format="ci8" Width="8" Height="8" Offset="0x5580" TlutOffset="0x43C0"/>
+
+      
+        <!-- Adult Ruto Limb DisplayLists -->
+        <DList Name="gAdultRutoLeftThighDL" Offset="0xB8E0"/>
+        <DList Name="gAdultRutoLeftLegDL" Offset="0xBE38"/>
+        <DList Name="gAdultRutoLeftFootDL" Offset="0xC228"/>
+        <DList Name="gAdultRutoLeftTailDL" Offset="0xBD08"/>
+        <DList Name="gAdultRutoRightThighDL" Offset="0xAC20"/>
+        <DList Name="gAdultRutoRightLegDL" Offset="0xB178"/>
+        <DList Name="gAdultRutoRightFootDL" Offset="0xB568"/>
+        <DList Name="gAdultRutoRightTailDL" Offset="0xB048"/>
+        <DList Name="gAdultRutoTorsoDL" Offset="0x8D30"/>
+        <DList Name="gAdultRutoLeftUpperArmDL" Offset="0xA150"/>
+        <DList Name="gAdultRutoLeftUpperWingDL" Offset="0xAA20"/>
+        <DList Name="gAdultRutoLeftLowerWingDL" Offset="0xAB00"/>
```

Comment:
I'd probably call these forearm fins, wings are usually a distinct limb (usual meaning, not game meaning) rather than an add-on to a preexisting one.